packages/odyssey-react-mui/src/icons.generated/index.ts

```diff
@@ -87,3 +87,4 @@ export * from "./User";
 export * from "./Video";
 export * from "./Warning";
 export * from "./WarningFilled";
+export * from "./index";
```

Comment:
why do we need this export?

Reply:
Not sure how that got added. This file is auto-generated, but I'm still not sure why it included itself. I've removed it.

Reply:
6e7d1e70ca9f3a801ce5cc76700f276b735c1490